src/test/java/com/flightstats/hub/model/ChannelConfigTest.java

```diff
@@ -4,29 +4,25 @@
 import com.google.common.collect.Sets;
 import org.joda.time.DateTime;
 import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
 
 import static org.junit.Assert.*;
 
 public class ChannelConfigTest {
-    private static final Logger logger = LoggerFactory.getLogger(ChannelConfigTest.class);
```

Comment:
On a tangent, I am not 100% sure why we use a logger factory all over the place. Seems like there'd be some sort of annotation we could use.

Reply:
```@Slf4j```

Reply:
Ooh. I was afraid the reason we were doing this was that we weren't using that library. I'm absolutely going to start converting these. Thanks!

Reply:
I think it should work as there's some slf4j bridge to most loggers that seems to magically happen (presumably somehow via lombok or it's dependencies). You'd have to verify it though.

Reply:
There are a couple annotations in the codebase already (I think Zander started adding them before he left). I'll double check that they're working.

Reply:
Yep, they're working just fine.
As a side note, I'm going to start a confluence doc with common cleanup things we should try to do in the codebase as we touch stuff.

Reply:
👍